src/Str.php

```diff
@@ -77,6 +79,16 @@ public static function asTwigVariable(string $value): string
         return $value;
     }
 
+    public static function asLowerCamelCase(string $str): string
+    {
+        return lcfirst(self::asCamelCase($str));
+    }
+
+    public static function asCamelCase(string $str): string
+    {
+        return strtr(ucwords(strtr($str, ['_' => ' ', '.' => '_ ', '\\' => '_ '])), [' ' => '']);
```

Comment:
why transforming dots and backslashes to underscores here ?

Reply:
great question - I can't remember anymore where this came from :/. But it doesn't make any sense